### ISSUE TITLE
Validation images cannot be read properly even with high accuracy

### ISSUE THREAD

**Issue by IAmGreyBunny** [clovaai/deep-text-recognition-benchmark#300]
Hi, i am trying to train an ocr model for recognising license plate. I am currently training with None-VGG-BiLSTM-CTC to be used with easyocr. i have a training dataset of 800 and a validation dataset of 200 generated by a script which creates varying background and font color. During training, the validation accuracy goes up steadily to about 70% to 80% after 1000 iterations however when i used it in the demo.py or in easyocr on some of the images from the same validation dataset that is being used during training, it gives random result that does not even come close to the actual word. I am new to AI and machine learning in general, any help would be appreciated.

Things ive tried:
1) changing valInterval(increasing and decreasing)
2) batch-size(from 192 to 32)

Below is my opt.txt for the training

------------ Options -------------
exp_name: None-VGG-BiLSTM-CTC-Seed1111
train_data: result/train
valid_data: result/test
manualSeed: 1111
workers: 4
batch_size: 32
num_iter: 300000
valInterval: 50
saved_model: 
FT: False
adam: False
lr: 1
beta1: 0.9
rho: 0.95
eps: 1e-08
grad_clip: 5
baiduCTC: False
select_data: ['/']
batch_ratio: ['1']
total_data_usage_ratio: 1.0
batch_max_length: 34
imgH: 64
imgW: 600
rgb: False
character: 0123456789!"#$%&'()*+,-./:;<=>?@[\]^_`{|}~ €ABCDEFGHIJKLMNOPQRSTUVWXYZabcdefghijklmnopqrstuvwxyz
sensitive: False
PAD: False
data_filtering_off: False
Transformation: None
FeatureExtraction: VGG
SequenceModeling: BiLSTM
Prediction: CTC
num_fiducial: 20
input_channel: 1
output_channel: 256
hidden_size: 256
num_gpu: 1
num_class: 97
---------------------------------------

**Comment by ozlem-atiz** [comment]
hello .İ am try to training the licance plate . But I have a problem like you. did you manage

**Comment by IAmGreyBunny** [comment]
> hello .İ am try to training the licance plate . But I have a problem like you. did you manage

Hi, unfortunately ive moved on from the project, never found a solution
